### ISSUE TITLE
Performances issue/question on W530 ThinkPad

### ISSUE THREAD

**Issue by KidCoding** [Bumblebee-Project/Bumblebee#243]
Hello,

I installed bumblebee and use the 304.43 nvidia drivers. I was able to use the graphics with optirun. optirun glxspheres was running at +/- 200 fps instead of 60 fps for glxspheres without optirun. But I was not able to use external display (in a proper manner without having to launch a command in the terminal when I connect or disconnect a screen or a projector).

So I decided to disable Optimus and use only the graphic card (option discrete in the thinkpad BIOS). External screen are now automatically detected. But when I run glxspheres, I get only 60 fps.

To summarize, discrete option disables the chipset, so evereything goes through the GPU. Bumblebee and optirun force to do the processing/rendering in the GPU. So the result is supposed to be the same but I get 40% of the performance with the discrete option...

Any explanation??

FYI: Here is a forum discussion about ThinkPad W530 and grahics/screen issues.
http://ubuntuforums.org/showthread.php?t=2007485

KiD
